lib/v2/focustaiwan/templates/article.art

```diff
@@ -0,0 +1 @@
+<img src="{{ $image }}"><div>{{ description }}</div>
```

Comment:
img src is not right according to Vercel's output.

Reply:
😊